.github/bin/label_abandoned_prs.py

```diff
@@ -36,7 +36,13 @@
         if labels['name'] == 'Abandoned PR - Needs New Owner':
             already_abandoned = True
 
-    if tdiff > twelve_days and not already_abandoned and not already_warned:
+    for review in json_data['reviews']:
```

Comment:
What if multiple people review the PR and one person has changes requested but the last person clicks "Approved", making the variable approved = True. Should we then have the state be considered approved if the developer does not address the first person's requested changes?